emission/core/wrapper/pipelinestate.py

```diff
@@ -27,6 +27,7 @@ class PipelineStages(enum.Enum):
     USER_MODEL = 7
     RECOMMENDATION = 8
     OUTPUT_GEN = 9
+    CREATE_PLACE_OBJECTS = 17
```

Comment:
there is already a `CREATE_CONFIRMED_OBJECTS`. This is a confirmed object. So why do we need to create a new stage for it?